angular-drag-and-drop-lists.js

```diff
@@ -73,13 +76,35 @@ angular.module('dndLists', [])
         });
       }
 
+      // Get handleElement, if any.
+      var handleElement = element[0].querySelector('.dndHandle');
+      // If handleCheck is not set to true on mousedown, dragging event in stopped.
```

Comment:
I think this querySelector should probably be done on mousedown, so that the .dndHandle element can be added after the initial element is created. Also, the querySelector can be changed or expanded to include [dnd-handle] attribute.